index.html

```diff
@@ -0,0 +1,231 @@
+          /></a>
+        </li>
+        <li class="nav-list">
+          <a href="#"
+            ><img src="youtube4.png" alt="twitter link" class="social-icon"
+          /></a>
+        </li>
+      </ul>
+    </nav>
+
+    <!--second nav-->
+
+    <nav class="second-nav">
+      <input type="checkbox" class="toggler" />
+
+      <div class="left-section">
+        <a href="index.html" class="logo">
+          <img
+            a
+            href="index.html"
+            src="restaurant.png"
+            alt="/"
+            class="logo-img"
+          />
+        </a>
+      </div>
+
+      <h1 class="ice">ICE</h1>
+
+      <ul class="second-nav-items">
+        <li class="nav-list">
+          <a href="about.html" class="nav1">About</a>
+        </li>
+        <li class="nav-list">
+          <a href="#program" class="nav2" href="#">Program</a>
+        </li>
+        <li class="nav-list">
+          <a class="nav3" href="#">Join</a>
+        </li>
+        <li class="nav-list">
+          <a class="nav4" href="#">Sponsor</a>
+        </li>
+        <li class="nav-list">
+          <a class="nav5" href="#">News</a>
+        </li>
+        <li class="nav-list">
+          <a class="nav6" href="#">Campaign</a>
+        </li>
+      </ul>
+    </nav>
+
+    <section class="main-section">
+      <h2 class="headline">Welcome to the Institute of</h2>
+      <h3 class="hl2">Culinary Education</h3>
+      <h4 class="hl3">Ranked No.1 Culinary School in South America*</h4>
+
+      <p class="para1">
+        The Institute of Culinary Education is a world-class culinary school
+        located in the heart of New York City. We offer a variety of programs to
+        help you achieve your culinary goals, whether you’re looking to become a
+        professional chef, start a new career, or just expand your cooking
+        skills.
+      </p>
+      <div class="main-time">
+        <h3 class="main-sub-heading">Registration starts 2022.10.15(MON)</h3>
+        <a class="zoom" href="#">Register now for Zoom link</a>
+      </div>
+    </section>
+
+    <section class="main-section2">
+      <img src="chef.jpg" alt="chef" class="chef-img" />
+      <h2 class="headline">Welcome to the Institute of</h2>
+      <h3 class="hl2">Culinary Education</h3>
+      <h4 class="hl3">Ranked No.1 Culinary School in South America*</h4>
+
+      <p class="para2">
+        The Institute of Culinary Education is a world-class culinary school
+        located in the heart of Santiago, Chile. We offer a variety of programs
+        to help you achieve your culinary goals, whether you’re looking to
+        become a professional chef, start a new career, or just expand your
+        cooking skills.
+      </p>
+      <div class="main-time">
+        <h3 class="main-sub-heading">Registration starts 2022.10.15(MON)</h3>
+        <a class="zoom" href="#">Register now for Zoom link</a>
+      </div>
+    </section>
```

Comment:
- [ ] [OPTIONAL]  Good job so far 👍 You made a lot of effort to make the `main section`, but your design is completely different from [Behance's design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), that is, you did not follow the layout and other styling given in [Behance design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015). Please do this, below I have tried to highlight the differences between you and Behance design!

### Your design ❌ 

![image](https://user-images.githubusercontent.com/63915024/191830156-3d78b41b-1013-4a5c-9161-a113276e1623.png)

### [Behance Design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) ✔️ 

![image](https://user-images.githubusercontent.com/63915024/191830268-7c964d80-a712-43ba-b617-061ba766b386.png)